scripts/deploy/deployCrossChainVerifierWithRequests.ts

```diff
@@ -368,7 +367,7 @@ async function main() {
       },
       scope: [
         {
-          circuitId: CIRCUIT_ID_SIG_V2,
+          circuitId: CircuitId.AtomicQueryMTPV2OnChain,
```

Comment:
@AndriianChestnykh @daveroga  could you please double check this?  According to the `invokeRequestMetadataKYCAgeCredential_MTPV2` and comments there was intention to add MTP circuits instead of SIG, that's why I've changed from `CIRCUIT_ID_SIG_V2` to `CircuitId.AtomicQueryMTPV2OnChain`

Reply:
I think you are right